releases/v1.0.6

```diff
@@ -0,0 +1,22 @@
+Highlights
+
+* Moved TestWorkflowRule into temporal-testing-junit4 module, if you use it please consume this package instead of temporal-testing.
+* Added abitlity to do async completion without having to specify task token.
```

Comment:
Explain that it obeys the parallelism limit

Reply:
Updated